gnowsys-ndf/gnowsys_ndf/ndf/views/type_created.py

```diff
@@ -0,0 +1,425 @@
+            pos_rts_obj = e.object_type
+            name_pos_rst = []
+            for each in pos_rts_obj:
+                k = node_collection.one({'_id':ObjectId(each) })
+                # print k.name,k._id        
+                name_pos_rst.append(k.name)
+            pos_rts_object_dict = dict({'name':e.name ,'altnames':e.altnames, 'object_type':name_rst})
+            f_pos_rts_object_dict.append(pos_rts_object_dict)
+        # the code above returns a dict for the object display in template -- for possible rts
+
+
+
+
+        # pos_ats = node.get_possible_attributes(node_gs._id)
+        # print pos_ats , "??////////////////???/ \n\n\n\n"
+        
+    else:
+        st_name = " "
+        node_gs = node_collection.one({'$and':[{'_type': u'GSystemType'},{'name':st_name}]})
+        ats = node_gs.attribute_type_set
+        rts = node_gs.relation_type_set
+        pos_ats = gsys_node.get_possible_attributes(node_gs_id)
+        pos_rts = gsys_node.get_possible_relations(node_gs_id)
+
+
+
+    # print "\n\n\n\n\n\n\n"
+    key_ats = []
+    for e in ats:
+        # print e.name
+        key_ats.append(e.name)
+    for e in final_ats:
+        # print e.name
+        key_ats.append(e.name)
+
+    # print key_ats
+
+
+    key_rts = []
+    for e in rts:
+        # print e.name
+        key_rts.append(e.name)
+    for e in final_rts:
+        # print e.name 
+        key_rts.append(e.name)
+    # print key_rts
+
+
+
+
+    newdict = {}
+
+    # if node_id:
+    #     new_instance_type = node_collection.one({'_type': unicode(gs_sys), '_id': ObjectId(node_id)})
+    # else:
+
+    new_instance_type = eval("node_collection.collection"+"."+gs_sys)()
+
+
+
+    if request.method == "POST":
+        print request.POST
+        # request.POST.get('IDoftheobject to be fetched')
+
+        for key,value in gs_struc.items():
+            # print key , value,"\n"
+
+
+            if key == "name":
+                print "name"
+                if request.POST.get(key,""):
+                    new_instance_type[key] = unicode(request.POST.get(key,""))
+
+
+            if key == "altnames":
+                print "altnames"
+                if request.POST.get(key,""):
+                    new_instance_type[key] = unicode(request.POST.get(key,""))
+
+
+            if key == "attribute_set":
+                print "this attribute_set "
+
+                for e in key_ats:
+                    # print e
+                    for k in request.POST.get(e,"").split(","):
+                        print e,k ,"\n"
+                        # here e -- key name , k -- key value eg. e -- nussd_course_type, k -- General
+
+
+            if key == "relation_set":
+                print "this relation_set "
+
+                for e in key_rts:
+                    for k in request.POST.get(e,"").split(","):
+                        print e,k ,"\n"
+                        # here e -- key name , k -- key value eg. e -- event_coordinator , k -- Person
+
```

Comment:
It was removed while testing, I have added it back. The lines added are used to save the data fetched from the form.